src/components/form-field/radio-box/radio-box-option.tsx

```diff
@@ -1,83 +1,79 @@
 import { RadioGroup } from "@headlessui/react";
-import React from "react";
+import React, { Fragment } from "react";
 import { classNames } from "../../../util/class-names";
-import { RecommendationTag } from "../../recommendation-tag/recommendation-tag";
 
 export interface RadioBoxOptionProps {
     children: React.ReactNode;
     value: string;
     disabled?: boolean;
-    recommendationText?: string;
     className?: string;
 }
 
-export const radioBoxContainerStyles = {
-    base: "group relative flex items-center gap-3 rounded-lg bg-neutral-0 border p-4 border-neutral-300 hover:border-primary-500 hover:bg-primary-50",
-    checked: "border-primary-500 bg-primary-500 hover:bg-primary-500 hover:text-neutral-0",
+const radioBoxContainerStyles = {
+    base: "group relative flex items-center gap-3 rounded-lg bg-neutral-0 border p-4 border-neutral-300 hover:border-primary-600 hover:bg-primary-50 cursor-pointer focus:outline-none",
+    checked: "border-primary-600 bg-primary-600 hover:bg-primary-600 hover:text-neutral-0",
     disabled:
-        "border-neutral-400 bg-neutral-100 group-hover:border-neutral-400 group-hover:bg-neutral-100 hover:border-neutral-400 hover:bg-neutral-100",
+        "bg-neutral-100 group-hover:border-neutral-300 group-hover:bg-neutral-100 hover:border-neutral-300 hover:bg-neutral-100 cursor-not-allowed",
+    active: "ring-2 ring-primary-200",
 };
 
-export const radioBoxCircleStyles = {
-    base: "relative inline-block h-4 w-4 shrink-0 rounded-full bg-neutral-0 border border-neutral-400",
-    unchecked: "group-hover:border-primary-500 group-hover:bg-neutral-0",
-    disabled: "bg-neutral-100 group-hover:border-neutral-400 group-hover:bg-neutral-100",
+const radioBoxCircleStyles = {
+    base: "relative inline-block h-4 w-4 shrink-0 rounded-full bg-neutral-0 border border-neutral-300",
+    unchecked: "group-hover:border-primary-600 group-hover:bg-neutral-0",
+    checked: "border-transparent",
+    disabled:
+        "bg-neutral-300 border-neutral-300 group-hover:border-neutral-300 group-hover:bg-neutral-300",
 };
 
-export const RadioBoxOption = ({
-    children,
-    value,
-    disabled,
-    recommendationText,
-    className,
-}: RadioBoxOptionProps) => {
+const Title = ({ children }: { children: React.ReactNode }) => (
+    <h2 className="headline-500 text-neutral-900 group-[.is-checked]:text-neutral-0 group-[.is-disabled]:text-neutral-500">
```

Comment:
I’ve tested or tried this already, but unfortunately this doesn’t work for non-`option`-html-elements. The radio boxes needs to render div elements which aren’t supporting css check and active states (at least for groups). The <option/> element isn’t working here, because we need to render more complex children than just strings :-)